wordlist.txt

```diff
@@ -37,3 +37,4 @@ KZGProofs
 KZGCommitmentInclusionProof
 LMD
 fulu
+BPO
```

Comment:
this can be removed, was previous in comment for `maxItems` but removed in last commit